### PR TITLE
[Bugfix] Small fixes for inline feedback component

### PR DESCRIPTION
**Pull request by fde312**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) locally.
- [x] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context

This PR fixes some small issues found during a testing session.
### Description

- When there is no score set, the tutor cannot save the inline feedback (also sets the background to yellow )
- When you click now on the result label, it shows the correct line number
- Fix wording

### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration
3. Create a programming exercise, participate and assess
4. Try out the to delete the score in th input field and check whether you can click on save or not
5. Check result label dialogs, should now list the same row as displayed in the gutter
6. Delete a unreferenced feedback element and check for the delete dialog.
7. Play around and see if you can break something

### Test Coverage

Nothing changed

### Screenshots

Not able to click on save when the score is not defined. Background is then also yellow, when score not defined.
image

Result label shows the correct line number now:
image

Delete a unreferenced feedback show the following:
image

### CODE REVIEW

File: src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     [style.max-width]="'780px'"
     [class.alert-success]="feedback.credits! > 0"
     [class.alert-danger]="feedback.credits! < 0"
-    [class.alert-warning]="feedback.credits === 0"
+    [class.alert-warning]="feedback.credits === 0 || feedback.credits == null"
```
Comment (schultek) [inline src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html:6]: You can just do this:
```suggestion
    [class.alert-warning]="!feedback.credits"
```

**Review by schultek — DISMISSED** [review]
This fixes the described issues. I just have a little code improvement.

**Review by FrankeLukas — DISMISSED** [review]

File: src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html (hunk 1 above)
Comment (fde312) [reply src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-inline-feedback.component.html:6]: Yeah, you are right! Thanks

**Review by schultek — APPROVED** [review]
Re-approve

**Review by FrankeLukas — APPROVED** [review]
Re-approve.

**Review by krusche — APPROVED** [review]

**Review by krusche — APPROVED** [review]

**Pull request merged**